charts/csm-replication/crds/replicationcrds.all.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Isn't this file going to become out-of-date as [the one in the csm-replication repository updates](https://github.com/dell/csm-replication/blob/main/deploy/replicationcrds.all.yaml)? I'm worried about the files falling out-of-sync. 

An alternative implementation would pull the relevant files from this repository, but that might be messy or difficult to work with. I just don't know how to feel about duplicate files across repositories.

Reply:
Going forward, we will eventually maintain the helm charts in the central helm repository – https://github.com/dell/helm-charts only. All the driver and modules related helm charts will be maintained here and the helm chart directory in the respective modules/driver repositories will be deprecated and removed. Until then we will have to maybe duplicate the helm charts in both the repositories.